eventorganizer.api/src/main/java/com/event/organizer/api/appuser/AppUser.java

```diff
@@ -116,4 +116,6 @@ public boolean isCredentialsNonExpired() {
     public boolean isEnabled() {
         return enabled;
     }
+
+    public List<Event> getEvents() { return this.events; }
```

Comment:
You don't need this. You already have it by default from the the framework. If you go to the `Event` class, on top you will see annotations `getter` and `setter`, their job is to automatically create the `getEvents` and `setEvents` as well for all other properties in the class